frontend/src/component/common/FavoriteIconButton/FavoriteIconButton.tsx

```diff
@@ -1,50 +1,58 @@
 import React, { VFC } from 'react';
-import { Dialogue } from 'component/common/Dialogue/Dialogue';
-import useFeatureApi from 'hooks/api/actions/useFeatureApi/useFeatureApi';
-import useToast from 'hooks/useToast';
-import { formatUnknownError } from 'utils/formatUnknownError';
-import { IconButton } from '@mui/material';
+import { IconButton, SxProps, Theme } from '@mui/material';
 import { ConditionallyRender } from '../ConditionallyRender/ConditionallyRender';
 import {
     Star as StarIcon,
     StarBorder as StarBorderIcon,
 } from '@mui/icons-material';
+import { TooltipResolver } from '../TooltipResolver/TooltipResolver';
 
 interface IFavoriteIconButtonProps {
```

Comment:
Is it possible to get those props automatically e.g. typeof IconButton so that next time we touch the code we don't need to keep updating the props. I see there's a section about making custom component composable here: fetcherCb

Reply:
This comment applied to IFavoriteIconButtonProps